service/src/main/kotlin/fi/espoo/evaka/application/Application.kt

```diff
@@ -115,7 +115,7 @@ data class Attachment(
     val contentType: String,
     val updated: OffsetDateTime,
     val receivedAt: OffsetDateTime,
-    val type: AttachmentType,
+    val type: AttachmentType?,
```

Comment:
Should we add a new attachment type? And if we re-use the application's Attachment, should these data types be moved to `attachment` package?

Reply:
Added a new, separate Attachment type to the backend and changed `type` here to be non-nullable again.